homeassistant/components/tradfri/strings.json

```diff
@@ -12,11 +12,11 @@
     },
     "error": {
       "invalid_key": "Failed to register with provided key. If this keeps happening, try restarting the gateway.",
```

Comment:
Don't wanted to change that one because message is quite explicit here and I think might not be replaced with something too generic.